### PR TITLE
feat(conference): add avatar.jpg URL

### PR DESCRIPTION
**Pull request by drymarau**
This can be used to provide a custom icon for a specific conference
alias.

### CODE REVIEW

File: sdk-conference/src/main/kotlin/com/pexip/sdk/conference/infinity/internal/ThemeImpl.kt [hunk 1]
```diff
@@ -121,20 +125,21 @@ internal class ThemeImpl(
         is ElementResponse.Unknown -> null
     }
 
-    private fun InfinityService.ConferenceStep.availableLayouts(store: TokenStore) = store.asFlow()
+    private fun InfinityService.ConferenceStep.availableLayouts(store: TokenStore) = store.token
+        .take(1)
```
Comment (tpettersen) [inline sdk-conference/src/main/kotlin/com/pexip/sdk/conference/infinity/internal/ThemeImpl.kt:129]: Why these `take(1)`s? Do we have more than 1 token?

File: sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Theme.kt [hunk 2]
```diff
@@ -23,15 +23,23 @@ import kotlinx.coroutines.flow.StateFlow
  */
 public interface Theme {
 
+    /**
+     * A [StateFlow] of the logo URL
+     */
+    public val avatar: StateFlow<String>
+        get() = throw NotImplementedError()
+
```
Comment (tpettersen) [inline sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Theme.kt:31]: According to the API docs, it says 
```
Obtains the image to display to represent a conference participant or directory contact.
```

Is this right to put as a "logo" in the theme?

**Review by tpettersen — COMMENTED** [review]
A couple of questions before approving

File: sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Theme.kt (hunk 2 above)
Comment (drymarau) [reply sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Theme.kt:31]: You're looking at the avatar of a participant, but this is for a conference itself. I believe it's not documented.

File: sdk-conference/src/main/kotlin/com/pexip/sdk/conference/infinity/internal/ThemeImpl.kt (hunk 1 above)
Comment (drymarau) [reply sdk-conference/src/main/kotlin/com/pexip/sdk/conference/infinity/internal/ThemeImpl.kt:129]: It's a StateFlow so there will be emissions when the token refreshes. But the layouts don't need to be updated every minute, just once (or if we encounter an error)

File: sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Theme.kt (hunk 2 above)
Comment (tpettersen) [reply sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Theme.kt:31]: 😭 I just love undocumented APIs

**Review by tpettersen — APPROVED** [review]

File: sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Theme.kt (hunk 2 above)
Comment (drymarau) [reply sdk-conference/src/main/kotlin/com/pexip/sdk/conference/Theme.kt:31]: It's like a treasure hunt!

**Pull request merged**